### PR TITLE
Book detail VI redirect

### PR DESCRIPTION
**Pull request by mcolombo87**
## Summary

Back botton with icon.

## Screenshots

image

## Trello Card

https://trello.com/c/rb5xLij1/33-detalle-de-libro-vi-redirecci%C3%B3n

## Bloqued by

#15

### CODE REVIEW

File: maquetado/detail.html [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-    <meta name="author" content="Wolox">
-    <link rel="stylesheet"  type="text/css" href="application.css">
-  </head>
+<head>
+  <meta charset="utf-8">
+  <title>Books</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
```
Comment (erikankodelia) [inline maquetado/detail.html]: The changes in the head section are note necessary

File: maquetado/detail.html [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
   <body>
+    <div>
```
Comment (erikankodelia) [inline maquetado/detail.html]: This div is not necessary

File: maquetado/detail.html (hunk 1 above)
Comment (mcolombo87) [reply maquetado/detail.html]: Done

File: maquetado/detail.html (hunk 2 above)
Comment (mcolombo87) [reply maquetado/detail.html]: Done

**Pull request merged**